packages/aws-cdk-lib/aws-ecs/lib/log-drivers/aws-log-driver.ts

```diff
@@ -82,6 +82,16 @@ export interface AwsLogDriverProps {
    * @default - AwsLogDriverMode.BLOCKING
    */
   readonly mode?: AwsLogDriverMode;
+
+  /**
+   * When AwsLogDriverMode.NON_BLOCKING is configured, this parameter
+   * controls the size of the non-blocking buffer used to temporarily
+   * store messages. This parameter is not valid with
+   * AwsLogDriverMode.BLOCKING.
+   *
+   * @default - No default
```

Comment:
I think our default here needs to be `1m` since this is the documented default for `maxBufferSize`. See linked documentation - https://docs.aws.amazon.com/AmazonECS/latest/userguide/using_awslogs.html

Reply:
Updated

Reply:
But wait, I think we want our default to be undefined since the blocking mode is default mode which is incompatible with this option.  Does the `@default` key word here only impact the docs or the CDK code itself? When a user does not specify this option, we do not want it to be set.